cypress/helpers/index.js

```diff
@@ -1,51 +1,60 @@
 const buildTableApiPath = (tableName)=> {
-    const baseAPIUrl = Cypress.env("BASE_API_URL");
-     return `${baseAPIUrl}/now/table/${tableName}`;
+  const baseAPIUrl = Cypress.env("BASE_API_URL");
+  return `${baseAPIUrl}/now/table/${tableName}`;
 }
 
 
 const bootStrapAcquisitionPackageApi= ()=> {
 
-    const acquisitionPackageApiEndpoint = buildTableApiPath('x_g_dis_atat_acquisition_package');
-    cy.fixture("acquistionPackage").then((data) => {
-        cy.intercept('POST', acquisitionPackageApiEndpoint, {
-            statusCode: 201,
-            body: data,
-         });
+  const acquisitionPackageApiEndpoint = buildTableApiPath('x_g_dis_atat_acquisition_package');
+  cy.fixture("acquistionPackage").then((data) => {
+    cy.intercept('POST', acquisitionPackageApiEndpoint, {
+      statusCode: 201,
+      body: data,
     });
+  });
    
 }
 
 export function bootstrapMockApis(){
     
-    bootStrapAcquisitionPackageApi();
+  bootStrapAcquisitionPackageApi();
     
 }
 
 export const cleanText = (text) => {
-    return text.replace(/\n/g, '').replace(/\s\s+/g, ' ').trim();
+  return text.replace(/\n/g, '').replace(/\s\s+/g, ' ').trim();
 }
 
 export const colors = {
-    primary:"rgb(84, 68, 150)"
+  primary:"rgb(84, 68, 150)"
 }
 
 function randomStringFromSet(characters, length) {
-  return [...Array(length)].map(() => characters.charAt(Math.floor(Math.random() * characters.length))).join("");
+  return [...Array(length)].map(
+    () => characters.charAt(Math.floor(Math.random() * characters.length))).join("");

```

Comment:
This triggered a security hotspot on SonarCloud which I have marked as safe. The usage of `Math.random` is not for a cryptographic purpose, it is to generate test data and sample values. The usage is safe in-context.